src/key/key.js

```diff
@@ -690,13 +690,15 @@ class Key {
         await user.verifyAllCertifications(verificationKeys, date, config) :
         [{ keyID: primaryKey.getKeyID(), valid: await user.verify(date, config).catch(() => false) }];
 
-      results.push(...signatures.map(
-        signature => ({
-          userID: user.userID.userID,
-          keyID: signature.keyID,
-          valid: signature.valid
-        }))
-      );
+      if (user.userID !== null && user.userID.userID !== null) {
+        results.push(
+          ...signatures.map((signature) => ({
+            userID: user.userID.userID,
```

Comment:
For example, you could do something like:
```suggestion
            userID: user.userID ? user.userID.userID : null,
            userAttribute: user.userAttribute,
```
(untested, lmk if it doesn't work).

Reply:
Thank you very much for the help and for allowing me to contribute to the project.

I think it worked!

Code: 
```javascript
const pub = `$armored_key`;
const k = await openpgp.readKey({ armoredKey: pub });
const vk = await k.verifyAllUsers(k);
console.log(vk);
```
Output:
<img width="883" alt="image" src="https://user-images.githubusercontent.com/46025304/235041810-a463eb6a-81ac-4059-9df3-704014597f3d.png">

Reply:
Thanks, and no worries!

Looks good, except it says `valid: false` for both, but I assume that's a separate issue (maybe you're using the wrong key to verify or so)?

If you get it to work, could you add a test case, perhaps? Something like [this](https://github.com/openpgpjs/openpgpjs/blob/726ee55de41a2ca305fce4c12fcd24696869bbc4/test/general/key.js#L2491-L2521) but with a key that has a User Attribute?

Reply:
Thanks, I did something wrong with the verification. I have also added a test - thanks for sharing a reference test case.